### ISSUE TITLE
Question about tbox and pbox

### ISSUE THREAD

**Issue by wangsssky** [ultralytics/yolov5#815]
## ❔Question
`pxy = ps[:, :2].sigmoid() * 2. - 0.5
pwh = (ps[:, 2:4].sigmoid() * 2) ** 2 * anchors[i]`
I don't quite understand these two lines above. the pxy and pwh seems not in the same scale.

and also this in the build_targets function.
`tbox.append(torch.cat((gxy - gij, gwh), 1))` gxy-gij are the offsets to the grid points, gwh are the wh in different anchor scales, why they can be together as the box?
Can anyone help？

**Comment by cc-sketch** [comment]
> ## Question
> `pxy = ps[:, :2].sigmoid() * 2. - 0.5 pwh = (ps[:, 2:4].sigmoid() * 2) ** 2 * anchors[i]`
> I don't quite understand these two lines above. the pxy and pwh seems not in the same scale.
> 
> and also this in the build_targets function.
> `tbox.append(torch.cat((gxy - gij, gwh), 1))` gxy-gij are the offsets to the grid points, gwh are the wh in different anchor scales, why they can be together as the box?
> Can anyone help？

hey, i also think that this is an error, and do you clear about it?
